feat(guidelines): clarify representation of multiple values for query parameters (#102)
otto-de/api-guidelines: api-guidelines/rest/resources/naming-conventions/rules/must-support-lists-for-multiple-values-of-the-same-parameter.md

# MUST support lists for multiple values of the same query parameter

> If a query parameter includes multiple values, provide them as a comma-separated list, for example, `key=value1,value2`).

Reviewer: ```suggestion
If a query parameter includes multiple values, provide them as a comma-separated list, for example, `key=value1,value2`.
```

# MUST support lists for multiple values of the same query parameter

> Represent multiple values of the same query parameter as a single comma-separated list, for example `key=value1,value2`.

Reviewer: ```suggestion
If a query parameter includes multiple values, provide them as a comma-separated list, for example, `key=value1,value2`).
Do not represent multiple values by repeating the same query parameter, for example, `key=value1&key=value2`.
```

> In OpenAPI, `key=value1,value2` can be represented as an array query parameter with `explode: false`.

Reviewer: ```suggestion
In OpenAPI, an array query parameter with `explode: false` also represents `key=value1,value2`:
```

```yaml
parameters:
  - in: query
    name: key
    explode: false
    schema:
      type: array
      items:
        type: string
```

::: warning
Don't represent multiple values by repeating the same query parameter, for example `key=value1&key=value2`.

> :::

Reviewer: ```suggestion
```
